gdrive

```diff
@@ -12,6 +12,8 @@ import sys
 import mimetypes
 from os.path import expanduser
 
+CHUNK_SIZE = 1024*256
```

Comment:
Maybe we should discuss to define how CHUNK_SIZE can affect our code.

Reply:
After a test with download function I decided to leave as 5MB since the default size was 100MB.